### ISSUE TITLE
Duplicate issues open when using multiple package files

### ISSUE THREAD

**Issue by OmgImAlexis** [renovatebot/renovate#2224]
**What Renovate type are you using?**
Renovate GitHub App

**Describe the bug**
Duplicate issues are opened for the same dependancy warning if you're using multiple package files.

**Additional context**
Please see https://github.com/pymedusa/Medusa/issues/4572

**Comment by rarkins** [comment]
Sorry for the noise caused, that was the result of a race condition I didn't anticipate - it's meant to check each time if a matching issue is already open however we check for updates from npm in parallel so each one created its own issue essentially simultaneously.

It only happens the first run, so is over now and shouldn't be repeated. However I'll disable the functionality on the bot until I've fixed it. I can perhaps combine it with #2225 as it looks like I need to delay the creation of issues until it can be done synchronously anyway.

**Comment by OmgImAlexis** [comment]
I know this is off-topic..

Is there anything else I need todo to get the bot running again now that we've merged the config into master? We had to change `baseBranches` and it looks like the bot's closed the old pin PR off because of that but nothing else has happened since.

**Comment by rarkins** [comment]
@OmgImAlexis it was just queued up - see https://github.com/pymedusa/Medusa/pull/4578
